buildSrc/src/test/kotlin/com/example/model/Animal.kt

```diff
@@ -22,7 +22,7 @@ public sealed class Animal {
 
     public data class Fish(
         public val water: Water,
-        public val size: Long? = null,
+        public val size: Long? = null
```

Comment:
where this change comes from? current `develop` is fine with trailing comma when running `ktlint` 0.50.0. I believe changes in `buildSrc` models should be reverted.

Reply:
I am not sure ...is for ktlint format...

Reply:
I'm running `ktlint` 0.50.0 and it doesn't do this change. Do you have the following version of `editorconfig` file https://github.com/DataDog/dd-sdk-android/blob/c49f70222b449d6edb8fbe05b92bba66ac89726f/.editorconfig in your branch, which is updated for `ktlint` 0.50.0?

Reply:
yeah I have the same...I need to run again the linter and see what happens, maybe I need to update the dd-source

Reply:
`ktlint` doesn't use files from `dd-source`